app/bitbox02-api.d.ts

```diff
@@ -37,10 +37,15 @@ declare module 'bitbox02-api' {
     prevOutIndex: number
   }
 
+  type CardanoAssetGroup = {
+    policyId: Uint8Array
+  }
```

Comment:
Seems like the "tokens" field is missing here

Reply:
not sure where that went ;) thanks for fixing it! feel free to merge it like this, your change looks good to me.

fyi, the next release v9.9.1 likely out next week, will support Byron recipient addresses (up to v9.9.0, only Shelley recipient addresses were supported).

Reply:
Will there any code changes in Adalite be needed to handle the Byron addresses with bitbox? I think not as we apparently didn't handle it anywhere so I guess it just causes the wallets with older firmware to crash, right? (We actually got a user's report related to that who later on contacted you too and got the issue solved)

Reply:
No changes needed, only a firmware upgrade - the BitBox02 is provided the address string and handles parsing, validating and cbor-serializing it.